### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by molker**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Please check if the PR fulfills these requirements
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

#### (Bug fix, feature, docs update, ...)

`[Type here]`

#### (You can link to an open issue)

`[Type here]`

#### (if this is a feature change)?

`[Type here]`

#### (What changes might users need to make in their application due to this PR?)

`[Type here]`

## Other information:

`[Type here]`

### CODE REVIEW

File: angular/README.md [hunk 1]
```diff
@@ -25,3 +40,36 @@ Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To u
...
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+# Ready to Integrate?
+
+## Building/exporting widget
+
+Once you are ready to export your widget, all you need is to run the following command in Terminal (Command line tool):
+
+```
+yarn build
```
Comment (cmullenx) [inline angular/README.md:51]: they can run one command ```yarn build:element``` to create the js file

File: angular/README.md [hunk 2]
```diff
@@ -25,3 +40,36 @@ Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To u
...
+yarn package
+```
+
+Running `build` will create a /dist/ folder in the root directory of our widget, and following up with `package` will create a single `index.js` file that contains the entire widget to be integrated. This file can be renamed and uploaded to a preferred location on a CDN (e.g. an S3 bucket on AWS. Please keep in mind tht this file has to be publicly available over the internet to be accessible to Agent or Supervisor Desktop).
```
Comment (cmullenx) [inline angular/README.md:55]: typo where it says please keep in mind tht, it should say that

File: angular/README.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
-# MyApp
+# WxCC Widget Starter - Angular
```
Comment (cmullenx) [inline angular/README.md:1]: i would keep the work in progress disclaimer that is in the current angular repo

File: angular/README.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
+Before starting, make sure you have yarn installed to be able to run the scripts in the `package.json` and are running node version `14.18.2` (this can be checked in your terminal with `node -v`).
+
+## Getting Started
+
+To run your widget on `localhost`, please navigate to widget's root directory in Terminal (Command line tool) and run the following commands (Assuming you have [`yarn`](https://classic.yarnpkg.com/en/docs/install/#mac-stable) installed globally on your machine):
+
+1. Clone this repo.
+2. Navigate to the **webex-contact-center-widget-starter/angular** folder.
+3. Run `yarn` or `yarn install` from the root of the repo.
+4. Run `ng serve` or `yarn start` for a development server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
```
Comment (cmullenx) [inline angular/README.md:22]: ```ng serve``` will only work if they have angular cli installed

File: angular/README.md [hunk 5]
```diff
@@ -1,10 +1,25 @@
...
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.11.
 
-## Development server
+This widget starter can be run on `localhost` to be reviewed. In case you (the developer) are using this widget starter as a foundation of your custom widget, running the sandbox environment on `localhost` would be a perfect way to build your widget's logic and interface.
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+**Note**: please keep in mind that we recommend to design every widget as responsive. With the exception of widgets for Agent or Supervisor Desktop **header** zone (where widgets should not exceed 64px height), you can assume that your custom widget might end up getting placed on various canvas sizes. Based on the Agent's screen size and configuration by the Administrator, it could even end up with a different aspect ratio. Please keep that in mind while testing your responsive widget's behavior.
+
+# Development
+
```
Comment (cmullenx) [inline angular/README.md:10]: I would add in some blurb in this section to let them know if they want to make edits to the widget they can do so in the /my-custom-component folder

File: angular/README.md (hunk 3 above)
Comment (molker) [reply angular/README.md:1]: I thought you had intentionally removed it, will add it back

File: angular/README.md (hunk 4 above)
Comment (molker) [reply angular/README.md:22]: I'll add angular cli to the prerequisites

**Review by cmullenx — APPROVED** [review]
LGTM

**Pull request merged**